src/apps/basic-transaction/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+// components
+import useBottomMenuModal from '../../hooks/useBottomMenuModal';
+import TextInput from '../../components/Form/TextInput';
+import Button from '../../components/Button';
+import FormGroup from '../../components/Form/FormGroup';
+import Label from '../../components/Form/Label';
+import Select from '../../components/Form/Select';
+
+// utils
+import { visibleChains } from '../../utils/blockchain';
+
+const App = () => {
+  const [t] = useTranslation();
+  const [destinationAddress, setDestinationAddress] = React.useState<string>('');
+  const [value, setValue] = React.useState<string>('');
+  const [data, setData] = React.useState<string>('');
+
+  const chainOptions = visibleChains.map((chain) => ({
+    id: `${chain.id}`,
+    title: chain.name,
+    value: chain.id,
+  }));
+
+  const [chainId, setChainId] = React.useState<number>(+chainOptions[0]?.value);
+  const { show } = useBottomMenuModal();
+
+  const send = () => {
+    if (!destinationAddress) return;
+    show({
```

Comment:
I'm just wondering... do you think that the `show` method naming here is obscure / too generic? I know we can rename the variable like `{ show: showSendConfirmation }` on instantiation but most developers won't do this... not a show stopper, just an observation.

Reply:
I tend to agree because I changed method name couple times already as I can't find the right one, yet. I think once we get new design done we can finalize this method name, wdyt?

Reply:
sounds good!